<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DashBoard</class>
 <widget class="QMainWindow" name="DashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="editar1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editar2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editar3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editar4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cerrar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar Sesion</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="foto3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>200</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>199</width>
      <height>59</height>
     </rect>
    </property>
    <property name="text">
     <string>USUARIOS EN LINEA:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="activos">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>209</width>
       <height>229</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
